src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -254,7 +257,7 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
                   </OverlayTrigger>
                 )}
 
-                {!isVenueUsingPartyMap && (
+                {!isVenueUsingPartyMap && venue && (
```

Comment:
I expect `isVenueUsingPartyMap` will already be `false` when this is rendered in the admin panel, but if you wanted to be extra explicit, you could add the `venue &&` part to that variable. Then you wouldn't need to have it down here as well.